scripts/delete_kibana_indices.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Set the default config file
+DEFAULT_CONFIG_FILE="./config/kibana.dev.yml"
+
+# Check if a config file is provided and use the default if not
+if [ "$#" -eq 1 ]; then
+    CONFIG_FILE="$1"
+else
+    CONFIG_FILE="$DEFAULT_CONFIG_FILE"
+fi
+
+# Parse the YAML file using grep and sed to extract the required values
+ELASTICSEARCH_HOSTS=$(grep -E '^elasticsearch.hosts: ' "$CONFIG_FILE" | sed 's/.*: //')
+ELASTICSEARCH_USERNAME=$(grep -E '^elasticsearch.username: ' "$CONFIG_FILE" | sed 's/.*: //')
+ELASTICSEARCH_PASSWORD=$(grep -E '^elasticsearch.password: ' "$CONFIG_FILE" | sed 's/.*: //')
+
+# Set default values if the extracted values are empty
+[ -z "$ELASTICSEARCH_HOSTS" ] && ELASTICSEARCH_HOSTS="http://localhost:9200"
+[ -z "$ELASTICSEARCH_USERNAME" ] && ELASTICSEARCH_USERNAME="system_indices_superuser"
+[ -z "$ELASTICSEARCH_PASSWORD" ] && ELASTICSEARCH_PASSWORD="changeme"
+
+# Get the list of indices from the _cat/indices API
+echo "Getting list of indices..."
+INDICES=$(curl -s -w "%{http_code}" -X GET "${ELASTICSEARCH_HOSTS}/_cat/indices/.kibana*?format=txt" \
```

Comment:
Super happy to see this change, I had these steps manually written to delete the indexes every time.

One question though, do we need to get all the indices for `.kibana` like this.

I was always querying - `http://localhost:9200/.kibana` which would give me a JSON object and the 1st key would be the `.kibana` index which needs to be deleted

Reply:
@achyutjhunjhunwala There's other `.kibana*` indices as well (like the one for the task manager) that I'm trying to delete.